pkg/utils/comparators/maps.go

```diff
@@ -0,0 +1,10 @@
+package comparators
+
+import (
+	"reflect"
+)
+
+//AreStringMapsSame compares two maps which are string key/value
+func AreStringMapsSame(lhs, rhs map[string]string) bool {
+	return reflect.DeepEqual(lhs, rhs)
```

Comment:
do we need to wrap this in a function?

Reply:
Only in the sense that its a package of comparators